.circleci/config.yml

```diff
@@ -145,22 +174,55 @@ commands:
               bundle exec fastlane update_carthage_commit
 
 jobs:
-  runtest:
-    <<: *base_job
+  run-test-ios:
```

Comment:
Actually what do you think about splitting this between iOS 14 and 15, like 12? Instead of running 2 in one.

Reply:
Ohhhh... I like that. I like that a lot!

Reply:
Done!